windows/setup.ps1

```diff
@@ -42,15 +42,18 @@ Check-Call { C:\Python37\python -m pip install -r requirements.txt  }
 Check-Call { C:\ProgramData\chocolatey\choco install git -y }
 Check-Call { C:\ProgramData\chocolatey\choco install 7zip -y }
 Check-Call { C:\ProgramData\chocolatey\choco install cmake -y }
-Check-Call { setx PATH "$($env:path);c:\Program Files\CMake\bin" }
 Check-Call { C:\ProgramData\chocolatey\choco install ninja -y }
 
 # Deps
 Check-Call { C:\Python37\python  windows_deps_headless_installer.py }
 
 # Other software
-#Check-Call { C:\ProgramData\chocolatey\choco install jom -y }
-#Check-Call { C:\ProgramData\chocolatey\choco install mingw -y }
+Check-Call { C:\ProgramData\chocolatey\choco install jom -y }
+Check-Call { C:\ProgramData\chocolatey\choco install mingw -y }
```

Comment:
where is this needed? for the build I didn't need it.

Reply:
The version of openblas we are using has dependencies on 2 or 3 files from mingw. From commit comment:

We need to install mingw for openblas dependencies (see Install the MXNet Package for Python section https://mxnet.apache.org/get_started/windows_setup )

Reply:
but it's working for me without them...